### DISCUSSION TITLE
HTML PROBLEMA CON EL EJEMPLO

### DISCUSSION THREAD

**Discussion by MONZONPUNTOEXE** [Launch-X-Latam/MisionFrontEnd#60] [General]
Estoy en la carpeta de HTML y a lo último no se ve el ejemplo

![image](https://user-images.githubusercontent.com/67812164/192843159-a8be4004-9607-41c5-a17c-d7d10a14ed67.png)

**Comment by JorgeDevmm** [comment]
indicar la ruta bro para verificar

**Comment by Cerecero** [comment]
Recuerda que vamos por pasos, aun estamos en el INTRO, seguramente cuando llegemos al HTML ya incluiran todo lo que falta.
